CMSgov/mint-app
# [EASI-3340] Share Model Plans with Users

# EASI-3340

## Changes and Description

- Updated gql schema
- Updated resolvers
- Preliminary FE updates to resolve build errors
- Updated Postman endpoint



## How to test this change

- Start the system as usual
- Create a Model Plan in Postman
- Share the Model Plan in Postman with both valid and invalid users
- Assert that the expected behavior happens
  - Validate that shared model plans send emails through MailCatcher

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
